src/components/card/card.component.js

```diff
@@ -25,13 +26,18 @@ const Card = ({
 
   const renderChildren = () => {
     return React.Children.map(children, (child, index) => {
+      const childElement =
+        typeof child === "string"
+          ? React.createElement(Typography, {}, child)
```

Comment:
To use `Typography` with specific styling we can use 
```js
<Card>
  <Typography {...style}>text</Typography>
</Card>
```

I proposed the most straightforward solution for the reported bug, but this problem is more complex, as I mentioned in `Additional context`. I decided to use `Typography` for the `createElement` function to be consistent with default text styling. But to make it possible to add style, we will need to add 8 more props to the tile component that will affect only this `Typography` element, so I'm not sure if it's worth it.

Reply:
@alzieba I've checked this part of the code and it looks that there's no need to pass `spacing` prop to all possible children components as only `CardRow` and `CardFooter` accept it.

That means that instead of creating a new `Typography` component, you could do just a simple condition like:

```js
if (child.type === CardRow || child.type === CardFooter) {
 ... do something with this child
} return child
```